### ISSUE TITLE
Restarting prometheus metrics task

### ISSUE THREAD

**Issue by thevops** [nucypher/nucypher#3466]
**Describe the Bug**
The application prints the following error every few seconds:

```
	Error during prometheus metrics collection: Traceback (most recent call last):	
  File "/usr/local/lib/python3.12/site-packages/twisted/internet/base.py", line 695, in run	
    self.mainLoop()	
  File "/usr/local/lib/python3.12/site-packages/twisted/internet/base.py", line 705, in mainLoop	
    self.runUntilCurrent()	
  File "/usr/local/lib/python3.12/site-packages/twisted/internet/base.py", line 1090, in runUntilCurrent	
    call.func(*call.args, **call.kw)	
  File "/usr/local/lib/python3.12/site-packages/twisted/internet/task.py", line 251, in __call__	
    d = maybeDeferred(self.f, *self.a, **self.kw)	
--- <exception caught here> ---	
  File "/usr/local/lib/python3.12/site-packages/twisted/internet/defer.py", line 209, in maybeDeferred	
    result = f(*args, **kwargs)	
  File "/usr/local/lib/python3.12/site-packages/nucypher/blockchain/eth/trackers/prometheus.py", line 14, in run	
    collector.collect()	
  File "/usr/local/lib/python3.12/site-packages/nucypher/utilities/prometheus/collector.py", line 55, in collect	
    self._collect_internal()	
  File "/usr/local/lib/python3.12/site-packages/nucypher/utilities/prometheus/collector.py", line 233, in _collect_internal	
    child_application_agent.is_operator_confirmed(	
  File "/usr/local/lib/python3.12/site-packages/nucypher/blockchain/eth/decorators.py", line 94, in wrapped	
    return func(*args, **kwargs)	
  File "/usr/local/lib/python3.12/site-packages/nucypher/blockchain/eth/agents.py", line 483, in is_operator_confirmed	
    staking_provider_info = self.staking_provider_info(staking_provider)	
  File "/usr/local/lib/python3.12/site-packages/nucypher/blockchain/eth/decorators.py", line 94, in wrapped	
    return func(*args, **kwargs)	
  File "/usr/local/lib/python3.12/site-packages/nucypher/blockchain/eth/agents.py", line 475, in staking_provider_info	
    return TACoChildApplicationAgent.StakingProviderInfo(*result)	
builtins.TypeError: StakingProviderInfo.__new__() takes 5 positional arguments but 7 were given

Restarting prometheus metrics task!
```

I also noticed that the metric `operator_matic_balance` is `0.0`.
I don't know when this has been happening. I just noticed the above today.

**System (please complete the following information):**
 - Nucypher Version: v7.2.0 (Docker image)

**Comment by theref** [comment]
There were changes made to one of the contracts and we didn't catch that it was used by Prometheus. This is fixed in https://github.com/nucypher/nucypher/pull/3431/commits/c4adf420b1edfad6b46032a23163a7c9d2d2f928 and will be released in v7.3

**Comment by sammy007** [comment]
> There were changes made to one of the contracts and we didn't catch that it was used by Prometheus. This is fixed in [c4adf42](https://github.com/nucypher/nucypher/commit/c4adf420b1edfad6b46032a23163a7c9d2d2f928) and will be released in v7.3

Why not make a point release with metrics fixed though?

**Comment by derekpierre** [comment]
> Why not make a point release with metrics fixed though?

We are careful with releases since there are typically multiple things that need to be added/fixed, not just one. In addition to better Prometheus support, other fixes are planned for 7.3.x.

Since there is a workaround for now, i.e. don't enable Prometheus, which is optional and only for metrics and does not affect the actual functionality of the nodes as it pertains to access control, the need for an interim release is reduced.

This issue should be fixed when 7.3.x is released.

**Comment by derekpierre** [comment]
Closed via #3479 .
